### PR TITLE
feat: disable refresh dbt on dbt project type None

### PR DESCRIPTION
**Pull request by rephus**
Closes:  #3460

### Description:

image

image

### CODE REVIEW

**Comment by rephus** [comment]
@TuringLovesDeathMetal any copy changes ?

**Comment by TuringLovesDeathMetal** [comment]
niiiice! maybe this bit? Just to tell them what they need to change to be able to _**not**_ run `lightdash deploy`?

> This project was created from the CLI. If you want to refresh dbt, you need to run `lightdash deploy` or update your dbt connection type to use the `refresh dbt` button. 

image

File: packages/frontend/src/components/RefreshDbtButton/index.tsx [hunk 1]
```diff
@@ -31,6 +33,39 @@ const RefreshDbtButton: FC<ComponentProps<typeof BigButton>> = (props) => {
     )
         return <div></div>;
 
+    if (
+        health.data?.mode === LightdashMode.CLOUD_BETA &&
```
Comment (ZeRego) [inline packages/frontend/src/components/RefreshDbtButton/index.tsx:37]: why the cloud beta check?

Comment (rephus) [reply packages/frontend/src/components/RefreshDbtButton/index.tsx:37]: just to ensure it is not triggered locally. our cloud is still on Beta according to the lightdash mode
image

Comment (rephus) [reply packages/frontend/src/components/RefreshDbtButton/index.tsx:37]: yeah, you're right, just tested it locally and it doesn't make sense.

**Pull request merged**